base/gmp.jl

```diff
@@ -912,7 +912,7 @@ module MPQ
 
 # Rational{BigInt}
 import .Base: unsafe_rational, __throw_rational_argerror_zero
-import ..GMP: BigInt, MPZ, Limb, isneg, libgmp
+import ..GMP: BigInt, MPZ, Limb, isnegative, libgmp
```

Comment:
I don't think you need to import `isnegative` now that's in `Base`?

Reply:
I'll need to check all presence of `isneg` and `ispos` first.